### PR TITLE
Task to Distribute Article Story to Authors Followers

### PR DESCRIPTION
**Pull request by patilanuja**
Resolves: #64 

This ticket will ensure to distribute article story to the author's followers.

### CODE REVIEW

File: packages/server/src/events/tasks.ts [hunk 1]
```diff
@@ -25,7 +25,49 @@ export async function CreateArticleStory(_email: string, articleId: number) {
...
+        },
+    });
+
+    const newArticleStoryDistribution = await prisma.articleStoryDistribution.createMany({
+        data: followerIds?.map((followerId) => ({
+            articleStoryId: articleStory?.id!,
+            userId: followerId!,
+            isSeen: false,
+            storyId: articleStory?.id!,
+            expiresAt: new Date(),
```
Comment (prasenjeet-symon) [inline packages/server/src/events/tasks.ts:65]: Must expire after 24 Hours from the article's published date.

**Review by prasenjeet-symon — CHANGES_REQUESTED** [review]
Please correct the pointed issue otherwise everything looks good.

File: packages/server/src/events/tasks.ts (hunk 1 above)
Comment (prasenjeet-symon) [reply packages/server/src/events/tasks.ts:65]: @patilanuja You can use date-fns.

Comment (patilanuja) [reply packages/server/src/events/tasks.ts]: @prasenjeet-symon 
Updated Expire time. Please have a look

File: packages/server/src/events/tasks.ts [hunk 2]
```diff
@@ -25,7 +25,51 @@ export async function CreateArticleStory(_email: string, articleId: number) {
...
+
+    const followerIds = followers?.followers.map((follower) => follower.followerId);
+
+    const articleStory = await prisma.articleStory.findUnique({
+        where: {
+            articleId: _articleId
+        },
+    });
+
+    const expirationTime = add(new Date(), { days: 1 }); // Calculate expiration time (1 day from now)
```
Comment (prasenjeet-symon) [inline packages/server/src/events/tasks.ts:59]: It should be dependent on the Article's published time but it is alright it will work for now.

**Review by prasenjeet-symon — APPROVED** [review]
Looks good , Thanks for your valuable contribution 🙏

**Pull request merged**